ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -226,6 +226,20 @@ open class BottomCommandingController: UIViewController {
         preconditionFailure("init(coder:) has not been implemented")
     }
 
+    /// Current rectangle of the view that represents the given hero item.
+    ///
+    /// - Parameter heroItem: A `CommandingItem` contained in `heroItems`.
+    /// - Returns: The current rectangle in the coordinate system of the receiver.
+    @objc public func rectFor(heroItem: CommandingItem) -> CGRect {
```

Comment:
Does it make sense to add a demo controller scenario using a tooltip to point to this rect?

Reply:
As far as I can see the fluent tooltip needs an anchor view and doesn't support anchor rects. I could show a simple popover instead.

Reply:
I guess my main point is to provide a test/demo scenario for the functionality that you're adding.

Reply:
Added the popover example code.

Reply:
Thank you for adding that example!